cmd/podman/shared/funcs.go

```diff
@@ -15,8 +16,20 @@ func GenerateCommand(command, imageName, name string) []string {
 		name = imageName
 	}
 	cmd := strings.Split(command, " ")
-	// Replace the first element of cmd with "/proc/self/exe"
-	newCommand = append(newCommand, "/proc/self/exe")
+
+	// Replace the first element of cmd with "/proc/self/exe" if "podman"
```

Comment:
I would prefer a check to see if cmd[0] exists, if it does, then we execute it,   If not, then it goes through this code path.

Is there a way to setup an alias before executing a command?

Reply:
Can you elaborate on that? I find it *very* unpredictable and hard to explain the behavior of runlabels to users. Especially in case of Podman, we could run into cases where we execute another version/binary of it.

Reply:
If I setup a container to run on special/script/podman or special/script/docker which has enhanced commands, then I would find it aggravating that there would be no way to run the script, other then to rename it to a non special name.